### PR TITLE
RHEL installation

### PR DESCRIPTION
**Pull request by lpasquali**
Added installation information for RedHat EL 7.x

### CODE REVIEW

File: install/advanced/core/index.rst [hunk 1]
```diff
@@ -1267,12 +1267,526 @@ Install and enable HTTPS secured connection through the Let's Encrypt provider
...
+        *OAuth2 Service Parameters*
+
+18. Using `letsencrypt`
+^^^^^^^^^^^^^^^^^^^^^^^
+
+In case you want to use letsencrypt free certificates, you should configure nginx accordingly:
+
+  https://certbot.eff.org/lets-encrypt/centosrhel7-nginx.html
+
+Comment out any ssl parameter in nginx and replace with the parameters and paths given by certbot
```
Comment (afabiani) [inline install/advanced/core/index.rst:1789]: Missing new line here

File: install/advanced/core/index.rst [hunk 2]
```diff
@@ -1267,12 +1267,526 @@ Install and enable HTTPS secured connection through the Let's Encrypt provider
...
+    cheaper-busyness-max = 70            ; Above this threshold, spawn new workers
+    cheaper-busyness-backlog-alert = 16  ; Spawn emergency workers if more than this many requests are waiting in the queue
+    cheaper-busyness-backlog-step = 2    ; How many emergency workers to create if there are too many requests in the queue
+    # daemonize = /var/log/uwsgi/geonode.log
+    # cron = -1 -1 -1 -1 -1 /usr/local/bin/python /usr/src/{{project_name}}/manage.py collect_metrics -n
+
+10. Modify /etc/nginx/nginx.conf
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are not using letsencrypt, you should put tour certificates in the paths
```
Comment (afabiani) [inline install/advanced/core/index.rst:1514]: Typo here: "tour"

File: install/advanced/core/index.rst [hunk 3]
```diff
@@ -1267,12 +1267,526 @@ Install and enable HTTPS secured connection through the Let's Encrypt provider
...
+
+    geonode localhost = (root) NOPASSWD: /usr/bin/geonode
+    geonode localhost = (root) NOPASSWD: /usr/bin/geonode_updateip
+
+Save to /etc/sudoers from temporary file and exit.
+
+4. Configure PostgreSQL 13
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You most likely want to change thhe password before applying the sql commands
```
Comment (afabiani) [inline install/advanced/core/index.rst:1329]: Typo here: "thhe"

File: install/advanced/core/index.rst [hunk 4]
```diff
@@ -1370,34 +1884,34 @@ Now the geonode will run on your windows.
 Postgresql Database setup
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In this section we are going to install setup ``PostgreSQL`` database on GeoNode. GeoNode uses the ``PostgreSQL 11`` database. 
+In this section we are going to install setup ``PostgreSQL`` database on GeoNode. GeoNode uses the ``PostgreSQL 11`` database.
```
Comment (afabiani) [inline install/advanced/core/index.rst:1887]: Version of PostgreSQL is outdated here

**Review by afabiani — CHANGES_REQUESTED** [review]
@lpasquali thanks, I found few typos. Can you please take a look?

File: install/advanced/core/index.rst (hunk 4 above)
Comment (lpasquali) [reply install/advanced/core/index.rst:1887]: > Version of PostgreSQL is outdated here

Nope, that's the windows section, enterprise db link points to 11

**Review by afabiani — APPROVED** [review]

**Pull request merged**